tests/integration/test_hyper_transformer.py

```diff
@@ -47,44 +47,6 @@ def _reverse_transform(self, data):
         return data.astype('datetime64')
 
 
-class DummyTransformerMultiColumn(BaseTransformer):
```

Comment:
Deleting these tests because we are dropping support for multi-column transformers.